dockstore-webservice/src/main/resources/migrations.1.16.0.xml

```diff
@@ -17,7 +17,7 @@
 
 <databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
                    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd"
-                   context="1.15.0">
+                   context="1.16.0">
```

Comment:
Hmmm, wonder if this means we will need to re-migrate qa

Reply:
> These changes do mean we'll have downtime when we deploy 1.16, right? The old web service couldn't run against the new schema? If yes, please update the 1.16 checklist.

I don't think there should be any downtime because the old DOI columns still exist (they're just deprecated in this PR)